tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>For pedagogical reasons, I suggest to split the model setup and the numerics setup into at least two different cells. That should also make it easier to differ between the core of the tutorial (numerics) and what should be considered boilerplate in this context (model). I found myself wanting to ask for much more documentation on the latter part, but then realized that is not the point here, though I did not fully manage to constrain myself.</p><p> </p><p>Put another way: I think it should be clear exactly what is needed to invoke a line search method for contact mechanics problems, and ideally it should be indicated what is needed to make this work as a global line search. Right now, some guesswork is needed to identify the relevant parts of the code. The impressive thing is, if I guess correctly, remarkably few lines are needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>